.travis_target.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -ev
```

Comment:
Wasn't really used.  I had it there just to make the infrequent segfaults with the initial MRI integration was implemented in libfacter easier to read.